src/sql/src/plan/hir.rs

```diff
@@ -3355,6 +3425,11 @@ impl HirScalarExpr {
         f(depth, self)
     }
 
+    /// Returns an _internal_ error if the expression contains
```

Comment:
Comment is incorrect, function returns an option.

Reply:
Ah, yes, copy-paste fail! Will fix in a follow-up PR.